src/HavenoDaemon.test.ts

```diff
@@ -703,6 +740,27 @@ test("Can complete a trade", async () => {
 
 // ------------------------------- HELPERS ------------------------------------
 
+/**
+ * Wait until the given predicate returns true, or the timeout exceeds.
+ */
+async function sleepUntil(predicate: () => boolean, timeoutMs: number): Promise<void> {
```

Comment:
I missed that this is unused and should be removed.

Reply:
Removed it